src/polymer/polymer2-element-scanner.ts

```diff
@@ -230,21 +232,22 @@ class ElementVisitor implements Visitor {
 
     const className = node.id && node.id.name;
     const element = new ScannedPolymerElement({
+      className,
       astNode: node,
       tagName: isValue,
-      className,
       description: (docs.description || '').trim(),
       events: esutil.getEventComments(node),
       sourceRange: this._document.sourceRangeForNode(node),
       properties: getProperties(node, this._document),
       methods: getMethods(node, this._document),
       superClass: this._getExtends(node, docs, warnings),
       mixins: jsdoc.getMixins(this._document, node, docs, warnings),
-      privacy: getOrInferPrivacy(className || '', docs, false), observers
+      privacy: getOrInferPrivacy(className || '', docs, false),  //
```

Comment:
Bah, at this point I would probably just surrender this and let clang do its thing. Someone will eventually find this and remove it, not knowing it's true purpose. Your call.